content/working_with/console/widgets/blueprints.md

```diff
@@ -129,6 +129,7 @@ See Settings section for details on how to turn on/off this feature.
 ## Settings
 
 * `Refresh time interval` - The time interval in which the widget’s data will be refreshed, in seconds. Default: 10 seconds
+* `List of fields to show` - The fileds that presented in the table and the catalog.
```

Comment:
```suggestion
* `List of fields to show` - The fields that are presented in the blueprints list. Default: Created, Updated, Creator, State, Deployments
```

Reply:
I think that the default are: Creator, State, Deployments.
When I click on "Reset Templates" those 3 fields are shown. And the task itself is to limit to those three.

Reply:
I changed to:
```
* `List of fields to show` - The fields that are presented in the blueprints list. Default: Creator, State, Deployments.
```
[30a079d](https://github.com/cloudify-cosmo/docs.getcloudify.org/pull/2137/commits/30a079d7a8a627f00e03aeabbd4bb65a66e1079d)

Is it okay?

Reply:
No, you are talking about default for Blueprints page. And here we are talking about default for Blueprints widget.

Reply:
I updated it: [cc9bc7b](https://github.com/cloudify-cosmo/docs.getcloudify.org/pull/2137/commits/cc9bc7b449fea83173573a2639d78e27decfdc59)